src/FunctionControlPanel.java

```diff
@@ -0,0 +1,199 @@
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.List;
+
+public class FunctionControlPanel extends JPanel {
+
+    Color[] Colors = {Color.RED, Color.BLUE, Color.YELLOW, Color.GREEN, Color.GRAY};
+    JComboBox<Color> colorPicker = new JComboBox<>(Colors);   //Combo Box to pick the Color of the Function
+    GridBagLayout InputLayout = new GridBagLayout();
+    GridBagConstraints gbc = new GridBagConstraints();
+    private int count;
+    private final List<FunctionControlPanel> functionControlPanelList;
+    private final JPanel functionInputPosition;
+    private final PolynomialFunctionControl polynomialFunctionControl;
+
+    JTextField functionField = new JTextField(20);  //Textfield for the Function
+
+    public FunctionControlPanel(int count, List<FunctionControlPanel> functionControlPanelList, JPanel functionInputPosition, PolynomialFunctionControl polynomialFunctionControl) {
+        this.functionControlPanelList = functionControlPanelList;
+        this.functionInputPosition = functionInputPosition;
+        this.count = count;
+        this.polynomialFunctionControl = polynomialFunctionControl;
+        this.setBackground(Color.blue);
```

Comment:
Same as red background color for control panel, should be changed before merging

Reply:
Done